examples/application_epg/main.tf

```diff
@@ -13,35 +13,149 @@ provider "aci" {
+  name      = "test_bd"
+}
+
+resource "aci_rest" "rest_qos_custom_pol" {
+  path       = "api/node/mo/${aci_tenant.terraform_tenant.id}/qoscustom-testpol.json"
+  class_name = "qosCustomPol"
+
+  content = {
+    "name" = "testpol"
+  }
+}
+
+resource "aci_contract" "rs_prov_contract" {
+  tenant_dn   = aci_tenant.terraform_tenant.id
+  name        = "rs_prov_contract"
+  description = "This contract is created by terraform ACI provider"
+  scope       = "tenant"
+  target_dscp = "VA"
+  prio        = "unspecified"
+}
+
+resource "aci_imported_contract" "rest_vz_cons_if" {
+  tenant_dn = aci_tenant.terraform_tenant.id
+  name      = "testcontract"
+}
+
+resource "aci_application_epg" "inherit_epg" {
   application_profile_dn = aci_application_profile.test_ap.id
-  name                   = "demo_epg"
-  description            = "from terraform"
-  annotation             = "tag_epg"
-  exception_tag          = "0"
-  flood_on_encap         = "disabled"
-  fwd_ctrl               = "none"
-  has_mcast_source       = "no"
-  is_attr_based_epg      = "no"
-  match_t                = "AtleastOne"
-  name_alias             = "alias_epg"
-  pc_enf_pref            = "unenforced"
-  pref_gr_memb           = "exclude"
-  prio                   = "unspecified"
-  shutdown               = "no"
+  name                   = "inherit_epg"
+  description            = "epg to create relation sec_inherited"
+  relation_fv_rs_node_att {
+    node_dn              = "topology/pod-1/node-108"
+    encap                = "vlan-100"
+    description          = "this is desc for static leaf"
+    deployment_immediacy = "lazy"
+    mode                 = "regular"
+  }
+}
+
+resource "aci_rest" "rest_qos_dpp_pol" {
+  path       = "api/node/mo/${aci_tenant.terraform_tenant.id}/qosdpppol-testqospol.json"
+  class_name = "qosDppPol"
+
+  content = {
+    "name" = "testqospol"
+  }
+}
+
+resource "aci_contract" "rs_cons_contract" {
+  tenant_dn   = aci_tenant.terraform_tenant.id
+  name        = "rs_cons_contract"
+  description = "This contract is created by terraform ACI provider"
+  scope       = "tenant"
+  target_dscp = "VA"
+  prio        = "unspecified"
+}
+
+resource "aci_rest" "rest_trust_ctrl_pol" {
+  path       = "api/node/mo/${aci_tenant.terraform_tenant.id}/trustctrlpol-testtrustpol.json"
+  class_name = "fhsTrustCtrlPol"
+
+  content = {
+    "name" = "testtrustpol"
+  }
+}
+
+resource "aci_taboo_contract" "rest_taboo_con" {
+  tenant_dn = aci_tenant.terraform_tenant.id
+  name      = "testcon"
+}
+
+resource "aci_monitoring_policy" "rest_mon_epg_pol" {
+  tenant_dn = aci_tenant.terraform_tenant.id
+  name      = "testpol"
+}
+
+resource "aci_contract" "intra_epg_contract" {
+  tenant_dn   = aci_tenant.terraform_tenant.id
+  name        = "intra_epg_contract"
+  description = "This contract is created by terraform ACI provider"
+  scope       = "tenant"
+  target_dscp = "VA"
+  prio        = "unspecified"
+}
+
+# data "aci_fabric_path_ep" "example" {
```

Comment:
Lionel wanted to keep these commented since the relation needs a static leaf to already exist. I think it's just there to show users how to implement it.